licenses/LICENSE-xz.txt

```diff
@@ -0,0 +1,11 @@
+Permission to use, copy, modify, and/or distribute this
```

Comment:
This comes from 
- https://github.com/tukaani-project/xz-java/blob/master/LICENSES/0BSD.txt